monitoring/bin/deploy_monitoring_viya.sh

```diff
@@ -7,6 +7,17 @@ cd "$(dirname $BASH_SOURCE)/../.."
 source monitoring/bin/common.sh
 source bin/openshift-include.sh
 
+# If openshift deployment, ensure user-workload monitoring is enabled
+if [ -n "$(kubectl get ns openshift-monitoring -o name 2>/dev/null)" ]; then
+  if [ -n "$(kubectl get po -n openshift-user-workload-monitoring -o name 2>/dev/null)" ]; then
+    log_debug "User workload monitoring is enabled"
+  else
+    log_error "Ensure that user workload monitoring is enabled"
+    log_error "openshift-user-workload-monitoring namespace was not found"
+    exit 1
```

Comment:
I wonder if "ERROR and exit" is the right behavior.  As it stands now, fixing the problem (i.e. updating the appropriate configMap) after running our script will fix the problem: there is no need to re-run the deploy_monitoring_viya.sh script.  Given that, maybe a WARNING is more appropriate? Although "ERROR and exit" should mean they can't miss the message.  So I can argue either side.  I also wonder if we should provide more information on how to "fix" the problem (e.g. tell them to do x-y-z, or, point them to the documentation)?